retail/interactive-tutorials/src/main/java/setup/SetupCleanup.java

```diff
@@ -0,0 +1,407 @@
+/*
```

Comment:
Just to double confirm: are both repositories using the same GCP project and environment variables for the sample tests? If not we may need to set them up in Cloud Console of the new projects following https://cloud.google.com/retail/docs/setting-up, and run the scripts to import the testing data.

Reply:
@averikitsch can you help me answer this question?

Reply:
This repo is using `java-docs-samples-testing` project according this setup:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.kokoro/tests/run_tests.sh#L58

And the `java-retail` repo is using the same project for the [samples - presubmit](https://github.com/googleapis/java-retail/blob/main/.kokoro/presubmit/samples.cfg#L17).  

Another project `gcloud-devel` is also used for [some other pre-submit tests](https://github.com/googleapis/java-retail/blob/main/.kokoro/presubmit/integration.cfg#L22). But not sure if that impacts this

Reply:
The other pre-submit tests (integration) are library specific and are separate from the sample. The sample is running on the same test project.

Reply:
Thank you Averi!